azure-pipelines/steps.yml

```diff
@@ -0,0 +1,32 @@
+steps:
+
+- task: UseRubyVersion@0
+  inputs:
+    versionSpec: '= 2.4'
+  condition: not(eq(variables['Agent.OS'], 'Darwin')) # not possible on Mac
```

Comment:
Can be this removed for now since only Windows build is triggered?

Reply:
Yep!

```suggestion
```